lte/protos/oai/mme_nas_state.proto

```diff
@@ -164,10 +164,11 @@ message UeContext {
 // mme_ue_context_t
 message MmeUeContext {
   map<uint64, uint64> imsi_ue_id_htbl = 4;  // imsi -> mme_ue_id
-  map<uint64, uint64> tun11_ue_id_htbl = 5; // tun_id -> mme_ue_id
+  map<uint64, uint64> tun11_ue_id_htbl = 5 [deprecated=true]; // tun_id -> mme_ue_id
```

Comment:
This is inline with the existing "deprecated" occurrences in this file. Can you point me to the coding guidelines for this? Because I will have to change all the occurrences if its mentioned in the guidelines